data/pc/latest/proto.yml

```diff
@@ -1542,7 +1531,40 @@
          if custom_report_details: packet_common_custom_report_details
          if server_links: packet_common_server_links
 ^play.toClient.types:
-   # World state information for spawn and respawn packets
+   packet_set_player_inventory:
```

Comment:
Please make sure to add up to date `# MC: ClassName` anchors to the top of all packet defs, this makes it easy to jump between the vanilla implementation and the protodef defs